Makefile

```diff
@@ -112,6 +112,18 @@ pause-container-release: pause-container
 	mkdir -p "$(shell pwd)/out"
 	@docker save ${PAUSE_CONTAINER_IMAGE}:${PAUSE_CONTAINER_TAG} > "$(shell pwd)/out/${PAUSE_CONTAINER_TARBALL}"
 
+# Variable to determine branch/tag of amazon-ecs-cni-plugins
+revision=master
```

Comment:
Hmm, this is alright but doesn't `cni-sources` implicitly convey a GET operation ?
From my prior experience, target names designed to fetch source code typically just have a `sources` target. 

I'm still in favor of retaining these target names as is.